File: lingting-huawei/src/main/java/live/lingting/framework/huawei/properties/HuaweiIamProperties.java
package live.lingting.framework.huawei.properties;

import live.lingting.framework.time.DatePattern;
import lombok.Getter;
import lombok.Setter;

import java.time.ZoneOffset;
import java.util.Map;

/**
 * @author lingting 2024-09-12 21:31
 */
@Getter
@Setter
public class HuaweiIamProperties {

	private String host = "iam.myhuaweicloud.com";

	private Map<String, Object> domain;

	private String username;

	private String password;

	private ZoneOffset zone = DatePattern.DEFAULT_ZONE_OFFSET;

}

File: lingting-huawei/src/main/kotlin/live/lingting/framework/huawei/properties/HuaweiIamProperties.kt
package live.lingting.framework.huawei.properties

import java.time.ZoneOffset
import live.lingting.framework.time.DatePattern

/**
 * @author lingting 2024-09-12 21:31
 */
class HuaweiIamProperties {
    var host: String = "iam.myhuaweicloud.com"

    var domain: Map<String, Any> = emptyMap()

    var username: String = ""

    var password: String = ""

    var zone: ZoneOffset = DatePattern.DEFAULT_ZONE_OFFSET
}
